README.md

```diff
@@ -41,6 +41,10 @@ Prompt tuning - learning soft prompts. This is different from prompt engineering
 
 The important difference between fine tuning and capabilities like prompt tuning/multi-taskprompt tuning is that the latter doesn't change the base model's weights at all. So when you run inference for prompt tuned models, you can have n prompts to 1 base model, and just inject the prompt tensors you need when they're requested instead of having _n_ separate fine-tuned models.
 
+### Benchmarking
+
+[Benchmarks](./benchmarks/README.md) for tuning various models.
```

Comment:
nit: Can we add a small description here too (similar to benchmarks readme) so that people are not looking for quality metrics when they look at this one.

Reply:
Rephrase as "Performance Benchmarking"?